client/my-sites/upgrades/components/form/hidden-input.jsx

```diff
@@ -37,14 +35,14 @@ module.exports = React.createClass( {
 	render: function() {
 		if ( this.state.toggled ) {
 			return (
-				<Input { ...this.props } initialFocus={ this.state.focus } />
-			);
-		} else {
-			return (
-				<div className="hidden-input">
-					<a href="" onClick={ this.handleClick }>{ this.props.text }</a>
-				</div>
+				<Input { ...this.props } autofocus />
 			);
 		}
+
+		return (
+			<div className="hidden-input">
+				<a href="" onClick={ this.handleClick }>{ this.props.text }</a>
+			</div>
+		);
```

Comment:
On a side note, why the hell are we calling this `HiddenInput`? I *always* assumed this was `<input type="hidden />`. We should change it's name to make more sense. (Out of scope for this)